analytics/src/analytics/metrics/burndown.py

```diff
@@ -62,10 +62,11 @@ def calculate(self) -> pd.DataFrame:
     def plot_results(self) -> Figure:
         """Plot the sprint burndown using a plotly line chart."""
         # Limit the data in the line chart to dates within the sprint
+        # or through today, if the sprint hasn't yet ended
         # NOTE: This will *not* affect the running totals on those days
         date_mask = self.results[self.date_col].between(
             self.dataset.sprint_start(self.sprint),
-            self.dataset.sprint_end(self.sprint),
+            min(self.dataset.sprint_end(self.sprint), pd.Timestamp.today(tz="utc")),
```

Comment:
Ok, this is a super nit pick that you can ignore, but I think in an ideal world, the graph would be easiest to use if the x-axis has a max of the end date of the sprint, but the *line* ends after today. So for current sprint, x-axis would run from Nov 29 until Dec 13th, but the line with the burndown would run from Nov 29 until Dec 5th (today). Does that make sense?

Reply:
Thanks for clarifying! I was able to update that in this commit: [fix: Adjusts x axis of burndown chart](https://github.com/HHS/simpler-grants-gov/pull/797/commits/8b2fef642e43cdb188d32f264ab1f2191419106a)